ember-cli-build.js

```diff
@@ -17,12 +17,14 @@ module.exports = function (defaults) {
     }
   })
 
-  /*
-    This build file specifes the options for the dummy test app of this
-    addon, located in `/tests/dummy`
-    This build file does *not* influence how the addon or the app using it
-    behave. You most likely want to be modifying `./index.js` or app's build file
-  */
+  if (app.env === 'test') {
```

Comment:
If so you'll probably find https://github.com/ciena-frost/ember-frost-bunsen/blob/master/blueprints/ember-frost-bunsen/index.js#L26-L34 useful.